versioning/scripts/dockerfiles/main.go

```diff
@@ -227,33 +230,41 @@ func check(e error) {
 }
 
 func main() {
-	templateDirPtr := flag.String("template_dir", "templates", "Path to directory containing Dockerfile.template and any other files to copy over")
+	defaultTemplateDirPtr := flag.String("template_dir", "templates", "Path to directory containing Dockerfile.template and any other files to copy over")
 	verifyPtr := flag.Bool("verify_only", false, "Verify dockerfiles")
+	var version = "0.01"
```

Comment:
I kind of understand what we want to achieve here. This however doesn't seem very useful as no one is gonna read this output. And it's not related to the change this PR is trying to make.

My suggestion is dropping this from this PR. We can open a separate PR that solves this concern better, for example `versions.yaml` can specify a required sem ver of the app.

Reply:
agreed; will remove it.